lambdas/authentication/register.py

```diff
@@ -58,6 +58,23 @@ def create_company_record(company_id, body):
         db_conn.commit()
     except Exception as e:
         raise RuntimeError(f"Error saving to company table: {e}")
+    
+    
+def create_position_record(position_id, company_id, position_name='Software Engineer'):
+    """
+    Creates a new position record in the database.
+
+    :param position_id: Unique identifier for the position.
+    :param company_id: Unique identifier for the company.
+    """
+    sql = """
```

Comment:
Don't think you need multiline for this:
sql = "INSERT INTO Position (id, company_id, name) VALUES (%s, %s, %s);"